generators/auth/templates/mongodb/jwt/src/utils/pagination.util.ts

```diff
@@ -0,0 +1,82 @@
+import { PaginationParamsInterface } from '@interfaces/pagination-params.interface';
+
+class PaginationUtil {
+  private static buildLink(location: string, paginationParams?: PaginationParamsInterface): string {
```

Comment:
think about how this function can be simplified by many small functions because I think this function contains too many logic